### ISSUE TITLE
Weird interface overlap

### ISSUE THREAD

**Issue by lentra0** [Knightmarevive/Knightmare-Kingdoms-clean#1]
Every time I install era 2 and this mod I have interface overlap in battle like there are two rows of buttons. Also hero button and next move button are bugged as well, double square normal buttons overlap with single square bugged buttons.

**Comment by Knightmarevive** [comment]
Yeah. Known Issue. there is place left for new buttons I use in future. I guess I may add help or heroesopedia or something :)
